main.go

```diff
@@ -23,6 +23,8 @@ import (
 	"io/ioutil"
 	"log"
 	"os"
+
+	"github.com/jbydeley/beacon/config"
```

Comment:
Yes, I should update it before merging and probably just branch off this repo directly next time. Good catch!